src/molecules/product-grid-item/ProductGridItem.scss

```diff
@@ -0,0 +1,132 @@
+@import '../../../assets/styles/_globals.scss';
+
+$catalog-grid-item__color                       : $gray-darker !default;
+$catalog-grid-item__font-size                   : $font-size-base !default;
+$catalog-grid-item__line-height                 : 1.5 !default;
+
+$catalog-grid-item__badge-top-position          : $spacer--medium !default;
+$catalog-grid-item__badge-left-position         : 0 !default;
+$catalog-grid-item__badge-z-index               : 1 !default;
+
+$catalog-grid-item__link-background-color       : #f9f9f9 !default;
+$catalog-grid-item__link-overflow               : hidden !default;
+$catalog-grid-item__link-white-space            : nowrap !default;
+$catalog-grid-item__link-text-overflow          : ellipsis !default;
+$catalog-grid-item__link-text-decoration        : none !default;
+
+$catalog-grid-item__image-default-opacity       : 0.85 !default;
+$catalog-grid-item__image-transform             : scale(1.1) !default;
+$catalog-grid-item__image-mix-blend-mode        : darken !default;
+$catalog-grid-item__image-transition            : $transition-base !default;
+
+$catalog-grid-item__details-background-color    : $white !default;
+$catalog-grid-item__details-padding             : $spacer !default;
+$catalog-grid-item__details-padding\@extra-large: $spacer--medium !default;
+
+$catalog-grid-item__name-color                  : $gray-darker !default;
+$catalog-grid-item__name-font-weight            : $font-weight-normal !default;
+$catalog-grid-item__name-margin                 : 0 0 $spacer 0 !default;
+
+$catalog-grid-item__price-margin                : $reset $reset $spacer $reset !default;
```

Comment:
In this case I am not sure if using `$reset` is more readable rather than `0` - like one line above.